base/loading.jl

```diff
@@ -151,42 +151,19 @@ function _include_from_serialized(path::String)
 end
 
 # returns an array of modules loaded, or an Exception that describes why it failed
-# and also attempts to load the same file across all nodes (if toplevel_node and myid() == master)
 # and it reconnects the Base.Docs.META
-function _require_from_serialized(node::Int, mod::Symbol, path_to_try::String, toplevel_load::Bool)
+function _require_from_serialized(mod::Symbol, path_to_try::String, toplevel_load::Bool)
     local restored = nothing
     local content::Vector{UInt8}
-    if toplevel_load && myid() == 1 && nprocs() > 1
-        # broadcast top-level import/using from node 1 (only)
-        if node == myid()
+    if myid() == 1
+        if toplevel_load
             content = open(read, path_to_try)
+            restored = _include_from_serialized(content)
         else
-            content = remotecall_fetch(open, node, read, path_to_try)
+            restored = _include_from_serialized(path_to_try)
```

Comment:
Can remove the `toplevel_load` bool and just always use this codepath. The other path is only relevant if it needed to also broadcast the file to all workers.